tests/dag_processing/test_job_runner.py

```diff
@@ -638,7 +639,7 @@ def test_scan_stale_dags(self):
         manager = DagProcessorJobRunner(
             job=Job(),
             processor=DagFileProcessorManager(
-                dag_directory="directory",
+                dag_directory=str(TEST_DAG_FOLDER),
```

Comment:
Why are we making this change? Makes it look like there's a problem

Reply:
Since we added a check that verifies the dag.fileloc is a descendant of DagFileProcessorManager's dag_directory path we need to modify the tests that had dummy paths as it will cause the tests to fail.